examples/todo/src/__tests__/acceptance/todo.acceptance.ts

```diff
@@ -87,7 +87,8 @@ describe('TodoApplication', () => {
   });
 
   it('creates an address-based reminder', async function() {
-    if (!available) return;
+    // eslint-disable-next-line no-invalid-this
+    if (!available) return this.skip();
```

Comment:
Good trick. I didn't know we can mark the test skipped this way. 

I was trying to use it.skip with the `available` flag calculated in the `before` hook but that is too late.